types/clamscan/clamscan-tests.ts

```diff
@@ -1,46 +1,40 @@
-import NodeClam = require("clamscan");
-import { PassThrough, Readable } from "stream";
-
-import axios from "axios";
+import * as NodeClam from "clamscan";
```

Comment:
Why is this import style changing in the test?

Reply:
A pure style preference

Reply:
I'm afraid they have different semantics. The export is `export =` and so should be `import =`. I am not sure why the CI isn't failing.